### PR TITLE
chore: update supported chains

### LINKED ISSUE #272: remove all goerli based testnets and all all the sepolia ones
new Sepolia L2 testnets:
* Mantle
* Optimism
* Arbitrum
* Base
* Blast

### PR DESCRIPTION
**Pull request by alanrsoares**
* upgrade viem, trpc, react-query
* update web3Modal connectors to prioritize injected wallets
* migrate goerli testnets to sepolia testnets

closes #272

### CODE REVIEW

**Comment by canhtrinh** [comment]
@alanrsoares what script do you run to determine which dependencies need to be updated? cc @npty

File: apps/maestro/src/config/evm-chains.ts [hunk 1]
```diff
@@ -196,7 +185,7 @@ export const ALL_CHAINS: ExtendedWagmiChainConfig[] = [
     environment: ENVIRONMENTS.mainnet,
   },
   {
-    ...optimismGoerli,
+    ...optimismSepolia,
     axelarChainId: "optimism",
```
Comment (canhtrinh) [inline apps/maestro/src/config/evm-chains.ts:189]: `optimism` -> `optimism-sepolia`: https://lcd-axelar-testnet.imperator.co/axelar/nexus/v1beta1/chains?status=1

File: apps/maestro/src/config/evm-chains.ts [hunk 2]
```diff
@@ -232,7 +221,7 @@ export const ALL_CHAINS: ExtendedWagmiChainConfig[] = [
     environment: ENVIRONMENTS.mainnet,
   },
   {
-    ...baseGoerli,
+    ...baseSepolia,
     axelarChainId: "base",
```
Comment (canhtrinh) [inline apps/maestro/src/config/evm-chains.ts:225]: `base` -> `base-sepolia`: https://lcd-axelar-testnet.imperator.co/axelar/nexus/v1beta1/chains?status=1

File: apps/maestro/src/config/evm-chains.ts [hunk 3]
```diff
@@ -274,49 +263,29 @@ export const ALL_CHAINS: ExtendedWagmiChainConfig[] = [
...
+  },
+  {
+    ...blast,
     axelarChainId: "blast",
     axelarChainName: "Blast",
     environment: ENVIRONMENTS.mainnet,
   },
+  {
+    ...blastSepolia,
+    axelarChainId: "blast",
```
Comment (canhtrinh) [inline apps/maestro/src/config/evm-chains.ts:285]: `blast` -> `blast-sepolia`: https://lcd-axelar-testnet.imperator.co/axelar/nexus/v1beta1/chains?status=1

**Review by npty — APPROVED** [review]

**Comment by alanrsoares** [comment]
> @alanrsoares what script do you run to determine which dependencies need to be updated? cc @npty

I usually run `npx @greenbot/cli`, but for this one particularly I just checked for specific deps:
- viem: for latest chain configs
- trpc (since we're using a prerelease)
- react-query (as a peer dependency of both wagmi and trpc)

**Comment by npty** [comment]
> @alanrsoares what script do you run to determine which dependencies need to be updated? cc @npty 

alan has built this awesome tool to do that. https://github.com/alanrsoares/greenbot so it might be this.

**Review by canhtrinh — APPROVED** [review]

**Pull request merged**